lib/circuitry/event_queue_creator.rb

```diff
@@ -0,0 +1,86 @@
+module Circuitry
+  class EventQueueCreator
+
+    attr_reader :sqs, :sns, :queue_name
+
+
+    def initialize(queue_name, sqs: Aws::SQS::Client.new, sns: Aws::SNS::Client.new)
+      @queue_name = queue_name
+      @sqs, @sns = sqs, sns
+    end
+
+    def verify_queue(topics)
```

Comment:
Maybe just me, but the word `verify` seems like a read-only method, similar to `validate`.  Let me think on it, but maybe there's another verb we can use for clarity.